save-backend/src/test/kotlin/org/cqfn/save/backend/controller/ExecutionControllerTest.kt

```diff
@@ -220,7 +220,7 @@ class ExecutionControllerTest {
         }
 
         mockServerPreprocessor.enqueue(
-            "/rerunExecution",
+            "/rerunExecution.*$",
```

Comment:
Do we really need `$` after `.*`? the `.*` should already match the remaining part of request string

Reply:
Well, `$` is there because we used to use `.containsMatchIn()` and now we moved to `.matches()` so now `$` is redundant

Reply:
Done